src/redisWrapper.js

```diff
@@ -128,23 +128,26 @@ const _createClientAndConnect = async (errorHandler) => {
   return client;
 };
 
-const _clientErrorHandlerBase = async (client, err, clientName) => {
-  _logErrorOnEvent(new VError({ name: VERROR_CLUSTER_NAME, cause: err, info: { clientName } }, "caught error event"));
-  if (client.isOpen) {
-    let quitError = null;
+const _closeClientBase = async (client) => {
+  if (client?.isOpen) {
     try {
       await client.quit();
     } catch (err) {
-      quitError = err;
-    }
-    if (quitError) {
-      _logErrorOnEvent(
-        new VError({ name: VERROR_CLUSTER_NAME, cause: quitError, info: { clientName } }, "error during client quit")
-      );
+      return err;
```

Comment:
why start such a syntax with quite errors here? Why not simply use the try...catch flow as in most places?

Reply:
That's a fair point. I will change that...